templates/src/main/scala/lucuma/schemas/ObservationDB.scala

```diff
@@ -16,27 +16,27 @@ import lucuma.core.math.BrightnessUnits._
 trait ObservationDB {
   object Scalars {
     // Ids
-    type AtomId              = Atom.Id
-    type ExecutionEventId    = ExecutionEvent.Id
-    type ObservationId       = Observation.Id
-    type ProgramId           = String
-    type StepId              = Step.Id
-    type TargetId            = Target.Id
-    type TargetEnvironmentId = TargetEnvironment.Id
+    type AtomId           = Atom.Id
+    type ExecutionEventId = ExecutionEvent.Id
+    type ObservationId    = Observation.Id
+    type ProgramId        = String
+    type StepId           = Step.Id
+    type TargetId         = Target.Id
+    type VisitId          = Visit.Id
```

Comment:
I'd lost track. I didn't even remember we had anything like a TargetEnvironment.Id anymore.

Reply:
At some point it was a thing.